### PR TITLE
Update dev walkthrough python and scene links

### PR DESCRIPTION
**Pull request by mwfarb**

### CODE REVIEW

**Comment by mwfarb** [comment]
@EdwardLu2018 Can you let me know if the Python updates to the library look right? @nampereira I know you are also making doc updates so I didn't want to change too much, just trying to help developers not get too confused. For now I just updated the `?scene=` style links to use the new scheme.

File: content/tutorials/dev-guide.md [hunk 1]
```diff
@@ -72,19 +70,19 @@ Controls:
...
+def on_msg_callback(obj):
+#    print("scene_callback: ", obj)
 
-def cube_callback(msg):
-    print("cube_callback: ", msg)
+def box_callback(evt):
+    print("box_callback: ", evt)
 
-arena.Object(objType=arena.Shape.cube, clickable=True, callback=cube_callback)
+box = Box(evt_handler=box_callback)
```
Comment (EdwardLu2018) [inline content/tutorials/dev-guide.md:82]: You still need `clickable=True` for evt_handlers.
Should be
`box = Box(clickable=True, evt_handler=box_callback)`

File: content/tutorials/dev-guide.md [hunk 2]
```diff
@@ -95,14 +93,24 @@ Now, in your scene use your mouse to click on the cube and notice the messages y
...
-             objName="duck_1",
-             location=(-1, 1, -3),
-             data='{"animation": { "property": "rotation", "to": "0 360 0", "loop": true, "dur": 10000}}',
-             url="models/Duck.glb")
-arena.handle_events()
+from arena import *
+
+scene = Scene(host="arena.andrew.cmu.edu", realm="realm", scene="example")
+
+obj = Model(objName="duck_1",
```
Comment (EdwardLu2018) [inline content/tutorials/dev-guide.md:100]: use `object_id` instead of `objName`

File: content/tutorials/dev-guide.md [hunk 3]
```diff
@@ -28,37 +21,42 @@ Our [User Guide](user-guide) will give you some pointers for navigation and vide
...
+Now, create the simplest Python program you can below, which will generate a 1-meter box. By default, objects are generated in a gray color, with no rotation, at scene x, y, z position (0, 0, 0), and with no other properties applied. Some of the other properties you can add to objects are detailed in our [Python Examples](../python/objects). Notice that the box seems stuck in the ground, which is due to the box's origin at its center positioned at scene coordinates (0, 0, 0). If you enable Flying mode (see [User Guide](user-guide)), you can move below the ground plane and view the other half of the box. Type Ctrl-C to end the program.
```
Comment (EdwardLu2018) [inline content/tutorials/dev-guide.md:26]: New aframe update randomizes colors, I believe. So objects are not gray by default if the color attribute is not specified in material.

File: content/tutorials/dev-guide.md (hunk 2 above)
Comment (EdwardLu2018) [reply content/tutorials/dev-guide.md:100]: Should be
`obj = GLTF(objName="duck_1",`

File: content/tutorials/dev-guide.md [hunk 4]
```diff
@@ -95,14 +93,24 @@ Now, in your scene use your mouse to click on the cube and notice the messages y
...
-             location=(-1, 1, -3),
-             data='{"animation": { "property": "rotation", "to": "0 360 0", "loop": true, "dur": 10000}}',
-             url="models/Duck.glb")
-arena.handle_events()
+from arena import *
+
+scene = Scene(host="arena.andrew.cmu.edu", realm="realm", scene="example")
+
+obj = Model(objName="duck_1",
+            location=(-1, 1, -3),
```
Comment (EdwardLu2018) [inline content/tutorials/dev-guide.md:101]: `location` was renamed to `position`

**Pull request merged**